### ISSUE TITLE
Game Doesn't Launch - After Clicking Run on ModOrganizer - Launches Graphics Screen_Then Closes

### ISSUE THREAD

**Issue by Ankmeister** [TitansBane/Elysium-Remastered#67]
I can't seem to figure out what is wrong:
Here are the Steps I'm following.
- I used Wabbajack(V2.5.3.24) to Download Elysium-Remastered (V1.9.2) via the Modlists Search
- Saved the installation of both to the root: C:\Wabbajack, and C:\Elysium Remastered
- Downloaded Mods via Elysium Remastered Launch Screen
- Installed Skyrim, SP on separate drive D:\, Then launched the game and installed all Creation Club and AE content.
- Finished Elysium Remastered with a "Installation Complete"
I followed all the instructions in the README.md, or at least I believe I have.
"Then, to start  run Elysium [SKSE] through Mod Organizer 2" located in the Elysium Remastered Installation folder.

Everything checks out?
But, when I Click Run in Mod Organizer 2.
The Graphics Splash screen launches "black screen with compile notes for shaders in the corner" and then closes returning to the Mod Organizer Screen.
Is there something wrong with one of the mods?
Or am I missing something?

Any help would be appreciated.
The Skyrim SE install is a a fresh vanilla install.

**Comment by vmekss** [comment]
Same problem here. Any success?

**Comment by dannem123** [comment]
Having the same issue as well

**Comment by dannem123** [comment]
Managed to get some help on the Discord server, it was an issue with my Antivirus (Bitdefender) so if you've also got the same antivirus try adding the following exceptions.
![bitdefenderelysiumfix](https://user-images.githubusercontent.com/52830039/187094346-101fb9a3-d099-4588-8455-9a7d48a42b60.png)

**Comment by Ghitubuser** [comment]
The antivirus didnt help because mine was already off. Is there anything else it could be

**Comment by WalterMPatrick** [comment]
> The antivirus didnt help because mine was already off. Is there anything else it could be

Check if in MO2 in Settings > paths > under the “managed game” is drive:\Elysium Remastered\Stock Game\SkyrimSE.exe

**Comment by Ghitubuser** [comment]
> in

Its what its supposed to be. Started up onto the colorful night sky then shut off

**Comment by Ghitubuser** [comment]
Got to the screen with the skill constellations but the mountains look almost overlapping bugged and then just crashes

**Comment by Stothe** [comment]
Same issue here with the game exiting after the splash screen with the skills constellations.  The log is reporting two errors:

[2022-10-19 02:03:56.183 E] failed to parse plugin file C:\Games\ElysiumRemastered\mods\Ancient Nord Stalhrim Patches (CCOR and Legacy of the Dragonborn Display)\DBM - Ancient Nord Stalhrim Patch.esp: file incomplete

[2022-10-19 02:15:16.564 E] invalid bsa 'C:/Games/ElysiumRemastered/mods/The Choice is Yours/TheChoiceIsYours.bsa', error can't read from bsa (filename: C:/Games/ElysiumRemastered/mods/The Choice is Yours/TheChoiceIsYours.bsa)

I turned off real time virus scanning as well.  I wanted to try turning off those two specific mods, but honestly I gave up on trying to find them.  If anyone knows what section they're under I can try that.

**Comment by Stothe** [comment]
I solved my particular problem.  After investigating the files mentioned in the error message, I found they were both zero bytes.  I deleted them and reran wabbajack which updated the files and it loaded.  Now I'm having a different issue where the title screen is only showing the upper left portion (the constellations, like it's rendering at 4x the size of my display).  But that's a different issue so I won't continue that in this issue.
